src/databaseConnection.ts

```diff
@@ -51,14 +47,45 @@ export async function databaseInit(): Promise<void> {
           game_round BIGINT DEFAULT 0 NOT NULL,
           starting_funds BIGINT NOT NULL,
           small_blind BIGINT NOT NULL,
-          small_blind_who SERIAL REFERENCES Players(id) NOT NULL,
+          small_blind_who VARCHAR(250) REFERENCES Players(token) NOT NULL,
           current_table_value BIGINT,
-          current_player SERIAL REFERENCES Players(id) NOT NULL
+          current_player VARCHAR(250) REFERENCES Players(token) NOT NULL
         )`
-      )
-    } catch (err) {
-      console.error(err)
-    }
+    )
+
+    await client.query(
+      `create or replace function insert_with_random_key (
+            game_master text,
+            card1 text,
+            card2 text,
+            card3 text,
+            card4 text,
+            card5 text,
+            game_round numeric,
+            starting_funds numeric,
+            small_blind numeric,
+            small_blind_who text,
+            current_table_value numeric,
+            current_player text)
+        returns numeric
+        language plpgsql
+        
+        as
+        $$
+        declare
+            game_id_rand numeric := CAST(random() * 1000000 AS INT);
+        begin
+            
+
+            insert into Games(game_id, game_master, card1, card2, card3, card4, card5, game_round, starting_funds, small_blind, small_blind_who, current_table_value, current_player) values (game_id_rand, game_master, card1, card2, card3, card4, card5, game_round, starting_funds, small_blind, small_blind_who, current_table_value, current_player) on conflict do nothing;
```

Comment:
Is it not possible to divide it into more lines? Almost 400 characters in one line doesn't sound good.